docker/hysds-io.json.topsstack-hamsar

```diff
@@ -5,28 +5,11 @@
   "action-type":  "both",
   "params": [
     {
-      "name": "min_lat",
+      "name": "bbox",
       "from": "submitter",
-      "type": "number",
-      "optional": true
-    },
-    {
-      "name": "max_lat",
-      "from": "submitter",
-      "type": "number",
-      "optional": true
-    },
-    {
-      "name": "min_lon",
-      "from": "submitter",
-      "type": "number",
-      "optional": true
-    },
-    {
-      "name": "max_lon",
-      "from": "submitter",
-      "type": "number",
-      "optional": true
+      "type": "region",
+      "optional": true,
+      "lambda":"lambda x: False if x == '' else region_to_bbox(x)"
     },
```

Comment:
Changes to allow ARIA system to automatically grab square bbox from user's AOI facet for stackSentinel.py input.